src/migrator.js

```diff
@@ -1,56 +1,64 @@
 // @flow
 
-import Command from 'cli-engine-command'
+import {type Config} from 'cli-engine-config'
+import Output from 'cli-engine-command/lib/output'
 import Plugins from './plugins'
 import path from 'path'
 import fs from 'fs-extra'
 
-export default class extends Command {
-  plugins: Plugins
+export default class {
+  static plugins: Plugins
+  static config: Config
+  static out: Output
 
-  async run () {
-    this.plugins = new Plugins(this.out)
-    let file = path.join(this.config.dataDir, 'plugins', 'plugins.json')
-    let pljson = await this.getPluginsJson(file)
+  static async run (plugins: Plugins, config: Config) {
```

Comment:
it's certainly fine to make this static, but it doesn't match the rest of the codebase well. I think you should have a constructor that takes in plugins and config, then a run method that takes no arguments and make everything instance not static methods